README.md

```diff
@@ -10,16 +11,18 @@ This has only been tested to work with React Native 0.47, probably works in earl
 ## Getting started
 
 ### Installing via RNPM (Common)
-```
-react-native link @synapsestudios/react-native-zendesk-support
+
+```bash
+react-native link @shiftsmartinc/react-native-zendesk-support
 ```
 
 ### Installing via Cocoapods (Not Common)
 
 Add the following line to your Podfile:
 
 ###### ios/Podfile
-```
+
+```bash
```

Comment:
The rest is mostly my IDE linter updating things and adding a language to some of our code examples.